control/nlsys.py

```diff
@@ -1663,87 +1671,190 @@ def ivp_rhs(t, x):
         success=soln.success, message=message)
 
 
-def find_eqpt(sys, x0, u0=None, y0=None, t=0, params=None,
-              iu=None, iy=None, ix=None, idx=None, dx0=None,
-              return_y=False, return_result=False):
-    """Find the equilibrium point for an input/output system.
+class OperatingPoint(object):
```

Comment:
Since we are using Python 3 and that only supports new classes, I think we should move to not inheriting from `object` (which is how things are done in the official documentation).